### DISCUSSION TITLE
YouTube and Daily Motion pretty much it?

### DISCUSSION THREAD

**Discussion by Logan82461** [aclap-dev/video-downloadhelper#649] [Help and Q&A]
### Your question

So really the only places to stream movies from are
Daily Motion, Youtube, and Tubi? 
Doesnt seem to work with any of the paid services and most of the others in the list provided for program are either sites not around anymore, questionable safety sites, or dont provide HD movies.

Are there others that I am unaware of you could please mention. I am mainly looking for old classics from the 30s through the 70s. 
Thanks

### Addon & CoApp version and details

not needed

**Comment by GrampaWildWilly** [comment]
This seems entirely off topic.  If you have a specific video you are trying to download but which VDH is failing on, post a proper problem report for that.  Otherwise, you really ought to take your question to a forum where they discuss classic movies.  That is not our focus here.

**Reply by Logan82461** [reply]
Yes and No to off topic. I could post dozens of  error codes I have received from numerous sites that I have tried to download videos from over the last few days, but I assume all the answers would be the same, , 
"VDH will not allow you to download videos from this site". 
. 
So my thought was to save some time for you and myself and inquire about sites that would work rather than overwhelm you with questions to verify that the sites I am trying are just not supported by VDH which I assume is the reason. 
I currently have only been able to find three legitimate and safe sites that this product seems to work with. Daily Motion, YouTube and Tubi. 

No need to answer further. I guess I have already answered my own question. VDH is just extremely limited in what it can do and if I wish for more, I need to look for other alternatives. Just wish I had realized this before paying for it.

**Reply by GrampaWildWilly** [reply]
I should make it clear that I do not work for VDH.  I am just another user.

VDH may be limited to just downloading videos.  That's all it was ever intended to do.  What more do you want it to do?  As for the license, it's a pittance.  And it gets cheaper by the minute seeing as it is for life.  It is not a subscription that you have to renew.  As long as you can breathe, the license will be yours to use on every computer you have now & every computer you may ever own in the future.  So how much you paid will become increasingly insignificant as time passes.

You would not overwhelm anybody by posting a single proper problem report.  A site on which VDH works fine is Medici TV.  It also works fine -- eventually -- on Pornhub & missav.  But listing every site where VDH does & does not work is not going to happen.  There are bazillions of web sites out there.  Bazillions.  That's a specialized geek term.  I hope you understand it.  Being a proud geek myself, I have no problem using the term.  So some of those bazillions of sites present no problem for VDH & some do.  See?  I just love saying bazillions.  If you pick a representative example, maybe I will be able to determine whether I can get VDH to work on it.  And if I can't, I do have an alternative technique that might turn up a different method that might work.  Might.  It doesn't always work.  But it does succeed a fair number of times.  It might be your answer on the other sites you won't mention.  So let's give it a shot -- ONCE -- & let's see where we get to.
